### PR TITLE
Unit tests for #1776

### PR DESCRIPTION
**Pull request by aschempp**
See the failing tests in the first commit, and merging #1776 fixes the issue.

### CODE REVIEW

File: core-bundle/src/Migration/Version409/CeAccessMigration.php [hunk 1]
```diff
@@ -77,6 +77,6 @@ public function run(): MigrationResult
             ':fields' => serialize(array_keys($GLOBALS['TL_FFL'])),
         ]);
 
-        return new MigrationResult(true, '');
+        return $this->createResult(true);
```
Comment (fritzmg) [inline core-bundle/src/Migration/Version409/CeAccessMigration.php:80]: This should not be part of this PR, should it?

Comment (aschempp) [reply core-bundle/src/Migration/Version409/CeAccessMigration.php:80]: that's because I upstream-merged 4.9 into master. It will be gone when rebasing once #1776 is merged.

**Review by ausi — DISMISSED** [review]

**Review by Toflar — DISMISSED** [review]

File: core-bundle/src/Routing/RouteProvider.php [hunk 2]
```diff
@@ -520,7 +526,7 @@ private function findRootPages(string $httpHost): array
             $rootPages = $pages->getModels();
         }
 
-        $pages = $pageModel->findBy(["tl_page.alias='index' OR tl_page.alias='/'"], null);
+        $pages = $pageModel->findBy(["tl_page.alias IN ('index', '/')"], null);
```
Comment (leofeyer) [inline core-bundle/src/Routing/RouteProvider.php]: What is the reason for this change?

Comment (leofeyer) [reply core-bundle/src/Routing/RouteProvider.php]: Ah, seems we have already discussed this in https://github.com/contao/contao/pull/1776/files#r432486188. 😄
